p2k16/roles/p2k16/tasks/main.yml

```diff
@@ -266,7 +266,9 @@
         version: master
       register: git_checkout
```

Comment:
hva med å registrere `p2k16_git_checkout` eller lignende for å unngå kollisjoner med andre roller?

Reply:
eller er registrerte variabler bare synline innenfor en _task_?

Reply:
Det hadde vært naturlig for meg, men jeg tviler på det egentlig. Får ta det som en egen jobb å undersøke. Kan bytte til p2k16_git_checkout for å være sikker for nå.